src/components/advanced-color-picker/advanced-color-picker.component.tsx

```diff
@@ -91,6 +98,13 @@ export const AdvancedColorPicker = ({
     };
   });
 
+  const currentAriaLabelColor = () => {
```

Comment:
this name is definitely wrong as the color is no longer used in the aria label. `currentSelectedColor` maybe?